ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Tell us what you'd like to see
+title: 'Feature Request: '
+labels: 'feature-request'
```

Comment:
Love the label adding. Do you know if there's a best practice around `feature-request` vs `feature request`? I ask because I actually created the one with spaces this week in the monorepo. Happy to go with either.

Reply:
I've definitely seen space-less names either hyphenated or titlecase as more common. But according to a quick GitHub search there are 50K issues on GitHub with a label `feature-request`, and 198K issues on GitHub with a label `feature request`.

It is definitely easier when constructing queries to use no spaces in the label names. An example of how to search for both:

With hyphen: `label:feature-request`
With space: `label:"feature request"`

So I think we should go with the hyphen.

Reply:
I ended up changing it back to the space in #7. Reason being is I think consistency within our own project is more important and I totally missed before that we use spaces consistently already. Thanks for pointing this out.

Reply:
I ended up changing it back to the space in #7. Reason being is I think consistency within our own project is more important and I totally missed before that we use spaces consistently already. Thanks for pointing this out.